DSCResources/SecurityPolicyResourceHelper/SecurityPolicyResourceHelper.psm1

```diff
@@ -96,6 +99,61 @@ function Invoke-Secedit
     Start-Process -FilePath secedit.exe -ArgumentList $arguments -RedirectStandardOutput $seceditOutput -NoNewWindow -Wait
 }
 
+function Get-SecurityPolicy

```

Comment:
This consolidates some of the Get-USRPolicy function into Get-UserRightsAssignment functions to be more specific in what it does.